docs/wiring/blower.rst

```diff
@@ -0,0 +1,43 @@
+.. _Blower Harness:
+
+Blower Harness
+======================
+
+.. csv-table:: **BOM**
+   :widths: 5, 60, 5, 5, 5, 10, 10, 10
+   :header-rows: 1
+   :delim: 0009
+   :file: ./blower.bom.tsv
+
+.. image:: blower.png
+  :width: 800
+  :alt: Wiring diagram
+
+You will need all the tools listed.
+
+* Using items from above BOM as well as:
+   * 3mm heat shrink tubing [:ref:`ew-shrink3`]
+   * 5mm heat shrink tubing [:ref:`ew-shrink5`]
+   * Heat gun
+   * Molex Micro-Fit 3.0 Crimper
+   * Wire Strippers
+   * Soldering iron
+   * Solder
+* Instructions:
+   #. Cut 3 pieces of the 5mm heat shrink, 15mm long each.
+   #. Thread the 5mm heat shrink onto the the entire bundle of 4 wires on the blower.  Do not shrink the heat shrink yet.
+   #. Insert the 2x pre-crimped Mega-Fit wires into the Megafit 2x2 housing.
+   #. Do a gentle pull test to confirm proper seating of the contacts.
+   #. Cut the cable at a length of 50mm from the connector.
```

Comment:
Is it easier to insert in housing and then cut and solder, or solder first and insert in housing later?

Reply:
I think the wires are long enough that either way is fine.  Did you have trouble with this?